src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -e
+
+BASE_URL=''
+DEVICE=''
+
+while getopts 'd:r:' flag; do
+  case "${flag}" in
+    d) DEVICE=$OPTARG;;
+    r) BASE_URL=$OPTARG;;
+  esac
+done
+
+DASH_DIR=`pwd`
+
+cd ../../../../build
+BUILD_DIR=`pwd`
+
+if [ "$BASE_URL" == "" ]; then
+    ../src/stop.sh
```

Comment:
when no vstart cluster is running, this command/script will block indefinitely. You should only stop the cluster in the end of the script, even if there was some error during the tests execution.

Reply:
Done